site/src/components/Calendar.vue

```diff
@@ -346,6 +353,16 @@ $dayFontSize: 0.8em;
     overflow-y: auto;
   }
 }
+
+div.calendar-event ::-webkit-scrollbar {
+  background: var(--sub-track-color);
```

Comment:
Where is this variable defined?

Reply:
line 48 of the same file. It sets those variable when it sets the background/border/text colors so that it's set per calendar element.

Reply:
ohhhh i didn't notice that, just a typo.